packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -30,6 +30,9 @@ const checkImageSource = require('./check.png');
 const uncheckImageSource = require('./uncheck.png');
 const mixedCheckboxImageSource = require('./mixed.png');
 const {createRef} = require('react');
+const fullImage = {
+  uri: 'https://www.facebook.com/ads/pics/successstories.png',
```

Comment:
Do you think we should reference a local asset here instead of fetching over network? I know there are some assets under `js/assets/search-icon.png`

Reply:
thanks a lot for the code review @lunaleaps. I added the changes required. More info in comment https://github.com/facebook/react-native/pull/31252#discussion_r624649351 :pray: